docs/README.md

```diff
@@ -60,6 +60,7 @@ All indicators can be updated over time by streaming data (prices or candles) to
 1. Stochastic RSI (STOCHRSI)
 1. True Range (TR)
 1. Wilder's Smoothed Moving Average (WSMA / WMA / WWS / SMMA / MEMA)
+1. Weighted Moving Average (WMA)
```

Comment:
Is this different to "Wilder's Smoothed Moving Average" (also shortened with "WMA")?

Reply:
Yup it's different. 

Weighted Moving Average is more commonly abbreviated to WMA, compared to Wilders Smoothed Moving Average which typically goes by WSMA or SMMA in my experience.